mkt/extensions/validation.py

```diff
@@ -70,10 +79,13 @@ def __init__(self, file_obj=None):
         self.file_obj = file_obj
         self.zipfile = None
 
-    def error(self, error_key):
+    def error(self, error_key, error_parameter=None):
```

Comment:
Or I guess we could use keyword arguments as well, to make the specification of multiple params in error messages localizations more flexible.